set.mm

```diff
@@ -123317,7 +123317,7 @@ distinguish between finite and infinite sets (and therefore if the set size
   1z $p |- 1 e. ZZ $=
     ( c1 1nn nnzi ) ABC $.
 
-  $( 1 is an integer, deductive form.  (Contributed by David A. Wheeler,
+  $( 1 is an integer, deduction form.  (Contributed by David A. Wheeler,
```

Comment:
You're right that things are not perfect. I tried to harmonize some nearby comments when I saw them, but nothing systematic.
I tried to be more systematic with:
* "is a complex number" instead of "is compex"
* "number xxx is real" instead of "number xxx is a real number"

As for the rest, it would be too long to harmonize "The number 1" versus "One". I tried to follow, in the cases I modified, the good practice that a sentence should not begin with a symbol.

Reply:
`good practice that a sentence should not begin with a symbol`
... except for the exception "1 is an integer...".  It's ok, I am not going to insist on this.

Reply:
> `good practice that a sentence should not begin with a symbol` ... except for the exception "1 is an integer...". It's ok, I am not going to insist on this. Actually, you already changed the line, I just noticed.

I did make the modification you suggested in commit b5ea15a